sdk/cosmos/azure-cosmos-kafka-connect/doc/configuration-reference.md

```diff
@@ -2,26 +2,36 @@
+| `kafka.connect.cosmos.throughputControl.account.azureEnvironment`         | `Azure`      | The azure environment of the CosmosDB account: `Azure`, `AzureChina`, `AzureUsGovernment`, `AzureGermany`.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
+| `kafka.connect.cosmos.throughputControl.account.tenantId`                 | `""`         | The tenantId of the CosmosDB account. Required for `ServicePrincipal` authentication.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
+| `kafka.connect.cosmos.throughputControl.auth.type`                        | `MasterKey`  | There are two auth types are supported currently: `MasterKey`(PrimaryReadWriteKeys, SecondReadWriteKeys, PrimaryReadOnlyKeys, SecondReadWriteKeys), `ServicePrincipal`                                                                                                                                                                                                                                                                                                                                                                                                                                                                         |
+| `kafka.connect.cosmos.throughputControl.accountKey`                       | `""`         | Cosmos DB Account Key                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
```

Comment:
```suggestion
| `kafka.connect.cosmos.throughputControl.accountKey`                       | `""`         | Cosmos DB Account Key (only required in case of auth.type as `MasterKey`)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         |
```

Reply:
updated